test/exercism/track_stream_test.rb

```diff
@@ -139,6 +139,53 @@ def test_read_unread_status
     assert clock.unread?, "clock should be unread"
   end
 
+  def test_watermark_is_read_when_greater_than_a_view
```

Comment:
D'OH.

I totally didn't realize that you had written a PR for this. I made the fix on the plane, and then submitted the PR first thing.